assets/css/codemirror/all.css

```diff
@@ -19,7 +19,7 @@
 }
 
 .CodeMirror-scrollbar-filler, .CodeMirror-gutter-filler {
-  background-color: transparent; /* The little square between H and V scrollbars */
+  background-color: white; /* The little square between H and V scrollbars */
```

Comment:
I don't know why this changed 😮

Reply:
This file is generated https://github.com/gravitee-io/gravitee-ui-components/blob/master/tasks/generate-theme.js#L44-L51

Reply:
Is it ok to update it in this PR or this I revert it?

Reply:
The PR retrieve the last fix of codemirror in 5.59.4 https://codemirror.net/doc/releases.html. 
I think it's okay, but maybe we should be careful and put strict versions on some dependencies ?